judy/src/App.js

```diff
@@ -0,0 +1,106 @@
+import './App.css';
+import React, { Component } from 'react';
+import Navbar from './components/Navbar/Navbar';
+import Intro from './components/Intro'
+import Work from './components/Work'
+import Play from './components/Play'
+import axios from 'axios'; 
+
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {currPage: "Home"}
+  }
+
+  changeCurrPage = (newCurrPage) => {
+    this.setState ({
+      currPage: newCurrPage
+    })
+  }
+  
+  renderPage = () => {
+    if(this.state.currPage === "Work") {
+      return <Work/>
+    }
+    else if(this.state.currPage === "Play") {
+      return <Play/>
+    }
+    else {
+      return <Intro/>
+    }
+  }
+ 
+  fetchData = () => {
+    // It will take an indeterminate amount of time for axios to get the data
+    // so we use this then/catch format
+    // this says: hit this endpoint. Then when you're done do X. 
+    // If there is an error at any point, do Y
+     axios.get("https://poetrydb.org/random")
+     .then((response) => {
```

Comment:
nice job working with the API. it could be fun to play with others too (there is so much free data out there :) )